lib/multiSigTask.ts

```diff
@@ -0,0 +1,385 @@
+    types.int
+  )
+  .addOptionalParam("isScheduled", "Check if proposal is scheduled | number", undefined, types.int)
+  .addOptionalParam("getTimestamp", "Get proposal timestamp | number", undefined, types.int)
+  .addOptionalParam(
+    "isProposalTimelockReached",
+    "Check if proposal time-lock has been reached | number",
+    undefined,
+    types.int
+  )
+  .addOptionalParam("isOwner", "Check if an address is an owner | string", undefined, types.string)
+  .addOptionalParam(
+    "isConfirmedBy",
+    "Check if proposal is confirmed owner | { proposalId:number, address:string }",
+    undefined,
+    types.json
+  )
+  .setAction(async (taskArgs, hre) => {
+    try {
+      console.log("running multiSig task");
+      const multiSigContract = await getContract(hre);
+      const signer = await getSigner(hre, taskArgs["signer"]);
+
+      if (taskArgs["getOwners"]) {
+        console.log("getting owners");
+        await hre.run("multiSig:getOwners", { contract: multiSigContract });
+      }
+
+      if (taskArgs["submitProposal"] !== undefined) {
+        console.log("submitting proposal");
+        await hre.run("multiSig:submitProposal", {
+          contract: multiSigContract,
+          destinations: taskArgs["submitProposal"]["destinations"],
+          values: taskArgs["submitProposal"]["values"],
+          payloads: taskArgs["submitProposal"]["payloads"],
+          signer: signer,
+        });
+      } else if (taskArgs["confirmProposal"] !== undefined) {
+        console.log("confirming proposal");
+        await hre.run("multiSig:confirmProposal", {
+          contract: multiSigContract,
+          proposalId: taskArgs["confirmProposal"],
+          signer: signer,
+        });
+      } else if (taskArgs["scheduleProposal"] !== undefined) {
+        console.log("scheduling Proposal");
+        await hre.run("multiSig:scheduleProposal", {
+          contract: multiSigContract,
+          proposalId: taskArgs["scheduleProposal"],
+          signer: signer,
+        });
+      } else if (taskArgs["revokeConfirmation"] !== undefined) {
+        console.log("revoking Confirmation");
+        await hre.run("multiSig:revokeConfirmation", {
+          contract: multiSigContract,
+          proposalId: taskArgs["revokeConfirmation"],
+          signer: signer,
+        });
+      } else if (taskArgs["executeProposal"] !== undefined) {
+        console.log("executing proposal");
+        await hre.run("multiSig:executeProposal", {
+          contract: multiSigContract,
+          proposalId: taskArgs["executeProposal"],
+          signer: signer,
+        });
+      } else if (taskArgs["getProposal"] !== undefined) {
+        console.log("getProposal");
+        await hre.run("multiSig:getProposal", {
+          contract: multiSigContract,
+          proposalId: taskArgs["getProposal"],
+        });
+      } else if (taskArgs["getConfirmations"] !== undefined) {
+        console.log("getConfirmations", taskArgs["getConfirmations"]);
```

Comment:
do we really need all the console.log statments under the else if statements?
kind of look like debug statements to me